lib/audit_tables/change_audit_table.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
+require_relative './base'
```

Comment:
Is this necessary to do in all these files? Is that something that could be done in the main `lib/audit_tables.rb` file ?